src/main/scala/io/iohk/ethereum/jsonrpc/TestService.scala

```diff
@@ -164,7 +164,7 @@ class TestService(
     accountAddresses = genesisData.alloc.keys.toList
     accountRangeOffset = 0
 
-    Task.now(Right(SetChainParamsResponse()))
+    SetChainParamsResponse().rightNow
```

Comment:
Being unfamiliar to this I find this worse, somehow... but it's purely subjective.

Reply:
Also, I'm kinda surprised that the core RPC methods aren't overloaded for 'bare' (non-`Task`) responses. Should be possible, right?